templates/web/greenwich/around/intro.html

```diff
@@ -1,6 +1,7 @@
 <h1>[% loc('Report a problem in Royal Greenwich streets or parks') %]</h1>
 <p>
-    Use this for reporting pysical problems that can be fixed like graffiti,
-    street cleaning issues or road defects. To report other kinds of problems
-    such as missed bins use our <a href="#">reporting forms</a>.
+    Use this for reporting pysical problems that can be fixed like graffiti,street cleaning issues or road defects. 
+</p>
+<p>
+    To report other kinds of problems such as missed bins use our <a href="#">reporting forms</a>.
```

Comment:
Whilst you're editing this copy, can you confirm what url we should use for this link? I found: https://www.royalgreenwich.gov.uk/site/xfp/scripts/xforms_form.php?formID=45&language=en, but is there a more generic page with other forms on it that we should use instead?